examples/generalTransferManager.ts

```diff
@@ -87,57 +89,63 @@ window.addEventListener('load', async () => {
     },
   });
 
-  // Change allow all whitelist transfers: I_GeneralTransferManager.modifyTransferRequirementsMulti(
-  //                 [0, 1, 2],
-  //                 [false, false, false],
-  //                 [false, false, false],
-  //                 [false, false, false],
-  //                 [false, false, false],
-  //                 { from: token_owner }
-//  await generalTM.changeAllowAllTransfers({ allowAllTransfers: true });
+  // Add owner address in the whitelist to allow mint tokens
+  await generalTM.modifyKYCData({
+    investor: myAddress,
+    canSendAfter: new Date(),
+    canReceiveAfter: new Date(),
+    expiryTime: new Date(2020, 0),
+    txData: {
+      from: await polymathAPI.getAccount(),
+    },
+  });
+
   const randomBeneficiary1 = '0x3444444444444444444444444444444444444444';
   const randomBeneficiary2 = '0x5544444444444444444444444444444444444444';
 
-  // Mint yourself some tokens and transfer them around, check balances
-  await tickerSecurityTokenInstance.issue({ investor: myAddress, value: new BigNumber(200), data: '' });
-  await tickerSecurityTokenInstance.transfer({ to: randomBeneficiary1, value: new BigNumber(50) });
-  console.log(await tickerSecurityTokenInstance.balanceOf({ owner: randomBeneficiary1 }));
-  console.log(await tickerSecurityTokenInstance.balanceOf({ owner: myAddress }));
-
-  // Disallow all transfers between token holders//
-//  await generalTM.changeAllowAllTransfers({ allowAllTransfers: false });
-
-  // Add whitelist special users
-  await generalTM.modifyKYCData({
-    investor: randomBeneficiary1,
-    canReceiveAfter: new Date(2018, 1),
-    canSendAfter: new Date(2018, 2),
-    expiryTime:  new Date(2035, 1),
+  // Add beneficiaries in the whitelist with distant dates to send and receive
+  await generalTM.modifyKYCDataMulti({
```

Comment:
The dates in this example should be relative to the current date, otherwise the example will behave differently after jan 2020

Reply:
Also, you should be able to add the owner's address to this transaction instead of doing it in a separate one

Reply:
1) ok
2) The idea was show the different methods we have to add someone to the whitelist, for that reason I decided to do it separately.

Reply:
That actually makes a lot of sense